exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/metric_exporter/__init__.py

```diff
@@ -91,6 +99,25 @@ def __init__(
             if insecure is not None:
                 insecure = insecure.lower() == "true"
 
+        if (
+            not insecure
+            and environ.get(OTEL_EXPORTER_OTLP_METRICS_CERTIFICATE) is not None
+        ):
+            credentials = _get_credentials(
+                credentials, OTEL_EXPORTER_OTLP_METRICS_CERTIFICATE
+            )
+
+        environ_timeout = environ.get(OTEL_EXPORTER_OTLP_METRICS_TIMEOUT)
```

Comment:
```suggestion
        environ_timeout = environ.get(OTEL_EXPORTER_OTLP_METRICS_TIMEOUT, 10)
```
or define constants for defaults?